packages/instant/webpack.config.js

```diff
@@ -6,27 +6,41 @@ const webpack = require('webpack');
 // The common js bundle (not this one) is built using tsc.
 // The umd bundle (this one) has a different entrypoint.
 
+const CDNS_THAT_REQUIRE_HEAP = ['production', 'staging', 'dogfood'];
```

Comment:
I wonder if `production` is kind of an overloaded term, because when the `NODE_ENV` is `production` the CDN could be `undefined` ( for CI and `yarn build` ) or the CDN could `production` (when discharging to production cdn). I wonder if it would make more sense to call this something like '0xproject.com' ?